CaptureTool/CaptureTool/Program.cs

```diff
@@ -14,9 +15,14 @@ static class Program
         [STAThread]
         static void Main()
         {
+            Thread readKeyThread = new Thread(ReadKeyThread.ReadKeys);
+            readKeyThread.Start();
```

Comment:
Probably, Preferences_Window_Load() would be better place to start readKeyThread.